### PR TITLE
Adding project to GitHub repo

### PR DESCRIPTION
**Pull request by RaliatB**
Function to take 2 numbers as parameters and add them . Then print out the result>

### CODE REVIEW

**Review by tofetpuzo — COMMENTED** [review]
I think you need a function not a statement

so something like this 

```
def sum_numbers(num1, num2):
      print(num1, num2)
```

File: raliat/addition.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+def sum_numbers(num1, num2):
+    sum = num1 + num2
+    print("Sum:", sum)
+    return sum
+result = sum_numbers(20, 20)
+
```
Comment (tofetpuzo) [inline raliat/addition.py:6]: `sum` is a key word in python can you please change it to another name. 

Then also you don't need the print("Sum:", sum) inside what you can do is this 

```
def sum_numbers(num1, num2):
    sum_nums = num1 + num2
    return sum_nums 
result = sum_numbers(20, 20)

print(result)
```

**Review by tofetpuzo — COMMENTED** [review]
here is a few issues I mentioned

**Comment by RaliatB** [comment]
I have edited the file.

**Pull request closed without merging**